llvm/utils/git/github-automation.py

```diff
@@ -45,6 +46,10 @@ def _get_curent_team(team_name, teams) -> Optional[github.Team.Team]:
             return team
     return None
 
+def escape_description(str):
+    # https://github.com/github/markup/issues/1168#issuecomment-494946168
+    str = html.escape(str, False)
+    return str.replace("@", "@<!-- -->")
```

Comment:
I guess this hides names completely.
Not that it matters much, but removing `@` seems a less intrusive way.

Reply:
Nah, it should keep everything @<!-- -->Endilll

Reply:
Ah, so you're inserting a dummy tag to break @ from name.
That's smart!